src/renderers/treeViewRenderer.js

```diff
@@ -0,0 +1,79 @@
+/* eslint no-console:0 */
+
+var _ = require('lodash');
+var randomColor = require('randomcolor');
+
+const supportedTypes = ["edu.illinois.cs.cogcomp.core.datastructures.textannotation.TreeLabelView"];
```

Comment:
This should be `edu.illinois.cs.cogcomp.core.datastructures.textannotation.TreeView`